opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py

```diff
@@ -193,7 +193,6 @@ def get_logger(
         schema_url: Optional[str] = None,
     ) -> Logger:
         """Returns a NoOpLogger."""
-        super().get_logger(name, version=version, schema_url=schema_url)
```

Comment:
I definitely agree with the change but is it related to this PR?

Reply:
Fixes the following mypy errors:

Call to abstract method "<method>" of "<class>" with trivial body
via super() is unsafe  [safe-super]

Reply:
Cool! I'm not sure why this code was like this (looks like it maybe got copy pasted). Maybe another reviewer will know why this was ever added, otherwise lgtm